### PR TITLE
Code cleanup

### PR DESCRIPTION
**Pull request by RaphealOlumofe**
Code cleanup issues

### CODE REVIEW

File: domain/src/main/java/starterProject/App.java [hunk 1]
```diff
@@ -15,19 +15,17 @@ public String getGreeting() {
     public static void main(String[] args) {
         createCar();
     }
-
-    private static List<Car> createCar() {
+    private static void createCar() {
```
Comment (petermcneil) [inline domain/src/main/java/starterProject/App.java]: This method doesn't really do much, why was it changed to be void?

File: frontend/src/main/java/olumofe/raphael/frontEnd/Service.java [hunk 2]
```diff
@@ -1,5 +1,9 @@
 package olumofe.raphael.frontEnd;
 
+public interface Service {
```
Comment (petermcneil) [inline frontend/src/main/java/olumofe/raphael/frontEnd/Service.java]: You're mixing up PRs, this PR should only have the codecleanup work in it.

File: frontend/src/main/resources/application.properties [hunk 3]
```diff
@@ -0,0 +1,3 @@
+spring.h2.console.enabled=true
```
Comment (petermcneil) [inline frontend/src/main/resources/application.properties]: Why is this here? There is no database atm

File: frontend/src/test/java/olumofe/raphael/frontEnd/CarControllerTest.java [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+    @Autowired
+    private TestRestTemplate restTemplate;
+
+   @Autowired
+    CarController carController;
+
+    @DisplayName("Test Spring @Autowired Integration")
+
+    @Test
+    public void checkObjectIsCreated() throws Exception {
```
Comment (petermcneil) [inline frontend/src/test/java/olumofe/raphael/frontEnd/CarControllerTest.java]: Similar to the comment above

**Review by petermcneil — CHANGES_REQUESTED** [review]

**Comment by IDWalsh** [comment]
There are still camel case package names.

File: frontend/src/main/resources/application.properties (hunk 3 above)
Comment (RaphealOlumofe) [reply frontend/src/main/resources/application.properties]: i was in the process of adding one, should i take i out for now

File: domain/src/main/java/starterProject/App.java (hunk 1 above)
Comment (RaphealOlumofe) [reply domain/src/main/java/starterProject/App.java]: I was getting a warning from inteliJ saying the return value is never used, and that i should make it a void, thus i made the method a void.

File: frontend/src/test/java/olumofe/raphael/frontEnd/CarControllerTest.java (hunk 4 above)
Comment (RaphealOlumofe) [reply frontend/src/test/java/olumofe/raphael/frontEnd/CarControllerTest.java]: The tests were written from when you asked me to write unit tests. No more test where done because Iain told me to stop writing them

File: frontend/src/main/resources/application.properties (hunk 3 above)
Comment (petermcneil) [reply frontend/src/main/resources/application.properties]: yes

File: frontend/src/test/java/olumofe/raphael/frontEnd/CarControllerTest.java (hunk 4 above)
Comment (petermcneil) [reply frontend/src/test/java/olumofe/raphael/frontEnd/CarControllerTest.java]: okay

**Review by petermcneil — APPROVED** [review]
